### PR TITLE
Standardize Readme (sans Title)

### PR DESCRIPTION
**Pull request by RichardLitt**
This standardizes the readme, without the contervsial title changes. See #577 for justification.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,13 +25,7 @@ This module saves you (and others!) time in two ways:
 - **Catch style errors before they're submitted in PRs.** Saves precious code
   review time by eliminating back-and-forth between maintainer and contributor.
 
-## Install
```
Comment (feross) [inline README.md]: Last nit: Can you move "Install" back here so it's above the TOC?

File: README.md [hunk 2]
```diff
@@ -518,10 +529,32 @@ var opts = {
...
 The `callback` will be called with an `Error` and `results` object (same as above).
 
+
 ## IRC channel
 
 Join us in `#standard` on freenode.
 
+## Contribute
+
+Contributions are welcome! Check out the [issues](https://github.com/feross/standard/issues) or the [PRs](https://github.com/feross/standard/pulls), and make your own if you want something that you don't see there.
```
Comment (feross) [inline README.md]: This is good -- thanks for adding it

**Comment by feross** [comment]
A few more comments inline

File: README.md (hunk 2 above)
Comment (RichardLitt) [inline README.md]: No problem. :)

**Comment by RichardLitt** [comment]
Moved to #577 again.

**Pull request closed without merging**